web/console/src/divisions/service.ts

```diff
@@ -27,6 +27,22 @@ export class DivisionsService {
 
         return divisionsSeasonsStatistics;
     }
+    /** handles gets divisions matches statistics */
+    public async getSeasonStatus(): Promise<any> {
```

Comment:
why any

Reply:
it was unnecessary code, so i deleted it